AWS/snapshot_recovery/script/restore.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
this script looks fine but please add it to a directory called `DR`, in https://github.com/mesoform/Multi-Cloud-Platform and call the script `vm_snapshot_recovery.sh`